src/components/datagridField/util.js

```diff
@@ -34,15 +34,17 @@ function chunkConditional(array, predicate) {
 }
 
 function buildVariableSizeFieldSection(columns) {
-  const filteredColumns = columns.filter((column) => (!column.meta || !column.meta.hidden));
-
   const groupedItems = [];
   let currentRowWidth = 0;
 
-  filteredColumns.forEach((column) => {
-    const fieldWidth = column.meta.width || 16;
+  columns.forEach((column) => {
+    const fieldWidth = column.meta.width;
 
-    if (column.type === "linebreak") {
+    if (column.meta.hidden && groupedItems.length === 0) {
+      groupedItems.push([column]);
+    } else if (column.meta.hidden) {
+      groupedItems[groupedItems.length - 1].push(column);
```

Comment:
Hidden field might take space on the UI. Please test this once with a large width hidden field to make sure there is no empty space on the UI

Reply:
Tested and verified that this does not impact the UI